src/uct/ib/rc/base/rc_iface.c

```diff
@@ -103,6 +103,7 @@ void uct_rc_iface_query(uct_rc_iface_t *iface, uct_iface_attr_t *iface_attr)
                                   UCT_IFACE_FLAG_CONNECT_TO_EP |
                                   UCT_IFACE_FLAG_AM_CB_SYNC |
                                   UCT_IFACE_FLAG_WAKEUP;
+    iface_attr->cap.align       = 256;
```

Comment:
can you check if it's possible to extract PCI MTU of the device (w/o need for root permissions)?